### ISSUE TITLE
Take Advantage of Accessibility without Paper Styles

### ISSUE THREAD

**Issue by Westbrook** [PolymerElements/paper-dialog-behavior#91]
### Description
Using `paper-dialog-behavior` for the accessibility benefits that it provides on top of `iron-overlay-behavior` currently requires a number of Paper Styles includes that don't seem to be required in the case of a functional behavior. Whether that means that those styles can be moved into "paper-dialog-shared-styles.html" or the accessible functionality can be moved elsewhere, I wanted to start a conversation on that here.

### Expected outcome
Accessible dialogs are possible without Paper Style includes.

### Actual outcome
Styles are included

### Live Demo
https://elements.polymer-project.org/elements/paper-dialog-behavior?view=demo:demo/index.html&active=Polymer.PaperDialogBehavior

### Steps to reproduce
1. Create an overlay with the paper-dialog-behavior

### Options
I had thought about just breaking a new version of this component for my local project, but I feel that this is something that might come up more and more as the community makes more broad use of the Polymer Team's elements, so I wanted to make sure I asked what you guys thought was most conducive to the sort of growth we all hope for around web components.

**Comment by valdrinkoshi** [comment]
Hi @Westbrook, I agree that `paper-dialog-behavior` has no reason for doing these imports:
```html
<link rel="import" href="../paper-styles/default-theme.html">
<link rel="import" href="../paper-styles/typography.html">
<link rel="import" href="../paper-styles/shadow.html">
```
These should be moved to `paper-dialog-shared-styles.html` instead.

The problem is that whoever is using `paper-dialog-common.css` is relying on the `paper-dialog-behavior` for these imports, so moving them would be a breaking change.
On the other hand, `paper-dialog-common.css` is deprecated...

@notwaldorf @cdata how should we go about this in your opinion?

**Comment by Westbrook** [comment]
I'd be happy to work up the PR for the above if indeed that was the right path forward for these features.

**Comment by cdata** [comment]
Yes, it is in danger of breaking users. But, it is a bug none-the-less (definitely not a feature) when we include imports that we don't need. It's also a bug (of the footgun variety) if users are depending on imports like this transitively. There is only one case where I know we do this deliberately (`paper-tabs` imports `paper-tab` so that users don't need to import both; this is a bad practice, but it was deemed an ergonomic win).

We should consider any PRs that remove unnecessary imports (if they are truly unnecessary), and do our best to assist users who are inadvertently broken by such a bug fix.

/cc @notwaldorf

**Comment by notwaldorf** [comment]
So, if I understand correctly, only the users of `paper-dialog-common.css` are affected, right? External CSS has been deemed bad since Polymer 1.1.0, and quite frankly it's a vulnerability vector to begin with, so if this update breaks you and forces you to use a `custom-style` include instead, I feel it's a net benefit.

Also, it has bothered me for a while that this behaviour is importing styles, since that's just bad design. I :+1: this

**Comment by valdrinkoshi** [comment]
Correct, only the users of `paper-dialog-common.css` would be affected, and indeed this would push them to upgrade to importing `paper-dialog-common.html` instead.

@Westbrook if you feel like it, please go ahead with a PR :)

**Comment by valdrinkoshi** [comment]
Merged and released in `v1.2.6` 🎉
